src/main/java/org/springframework/social/bitbucket/api/BitBucketEmailAddress.java

```diff
@@ -0,0 +1,54 @@
+/**
+ * Copyright (C) 2012 Eric Bottard / Guillaume Lederrey (eric.bottard+ghpublic@gmail.com / guillaume.lederrey@gmail.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.springframework.social.bitbucket.api;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.io.Serializable;
+
+/**
+ * BitBucket email address structure
+ *
+ * @author Cyprian Śniegota
+ * @since 2.0.0
+ */
+@JsonIgnoreProperties(ignoreUnknown = true)
+public final class BitBucketEmailAddress implements Serializable {
+
+    private static final long serialVersionUID = 1L;
+
+    @JsonProperty("active")
+    private Boolean active;
+
+    @JsonProperty("email")
+    private String email;
+
+    @JsonProperty("primary")
+    private Boolean primary;
+
+    public Boolean getActive() {
+        return active;
+    }
+
+    public String getEmail() {
+        return email;
+    }
+
+    public Boolean getPrimary() {
```

Comment:
This should probably by called `isPrimary()` and return a primitive `boolean` (not `Boolean`), unless there is a good reason for it to be `null` (in which case, that reason should be documented in a comment).